view/frontend/web/instantsearch.js

```diff
@@ -99,17 +100,6 @@ requirejs(['algoliaBundle', 'Magento_Catalog/js/price-utils'], function (algolia
 			routing: window.routing,
 		};
 
-		if (algoliaConfig.request.path.length > 0 && window.location.hash.indexOf('categories.level0') === -1) {
```

Comment:
hey @bsuravech note that in this PR I've removed `areCategoriesInFacets` case because I don't think it's still relevant, but please if you think otherwise let me know and I'll add it back.